src/earthengine/ee_worker.js

```diff
@@ -177,13 +173,6 @@ class EarthEngineWorker {
         // Run methods on image
         eeImage = applyMethods(eeImage, methods)
 
-        // Use mask operator (e.g. mask out values below a certain threshold)
-        if (maskOperator && eeImage[maskOperator]) {
```

Comment:
So we think masking data for zonal stats won't have use cases?
For NDVI, some authors remove negative values. Thinking of air pollution, it is not uncommon to have outliers. From the ERA5 precipitation description, I read it is possible to have negative values.

Reply:
Good question, I wanted to avoid first masking the image and the unmasking it again for performance reasons. Maybe we can change this behaviour later if a clear use case appear?

Reply:
Ok